llvm/docs/CodeOfConduct.rst

```diff
@@ -24,12 +24,12 @@ the spirit in which it's intended - a guide to make it easier to communicate
 and participate in the community.
 
 This code of conduct applies to all spaces managed by the LLVM project or The
-LLVM Foundation. This includes IRC channels, mailing lists, bug trackers, LLVM
-events such as the developer meetings and socials, and any other forums created
-by the project that the community uses for communication. It applies to all of
-your communication and conduct in these spaces, including emails, chats, things
-you say, slides, videos, posters, signs, or even t-shirts you display in these
-spaces. 
+LLVM Foundation. This includes Discord channels, mailing lists, bug trackers,
```

Comment:
Regarding the comment on not explicitly mentioning anymore that the code of conduct continues to apply to the IRC channel too: Probably the easiest way to address that is to keep IRC in the enumerated list here.
IRC is a "legacy" channel, but it still exists, so the CoC continues to apply there.